packages/story-editor/src/components/panels/design/videoOptions/videoOptions.js

```diff
@@ -136,8 +152,16 @@ function VideoOptionsPanel({ selectedElements, pushUpdate }) {
       clearEditing();
     } else {
       setEditingElementWithState(selectedElements[0].id, { isTrimming: true });
+      trimExistingVideo({ resource, start: '00:00:01', end: '00:00:02' });
```

Comment:
This is a test implementation for now. I will leave it to @barklund to do his thing to hook up these pieces.